domains/anonymousdeveloper.is-cool.json

```diff
@@ -0,0 +1,17 @@
+{
+    "description": "Portfolio website of Noah.",
+    "domain": "is-cool.dev",
+    "subdomain": "anonymousdeveloper",
+
+    "owner": {
+        "repo": "https://github.com/noah-dev-codes/noahcoding",
+        "email": "ak.no@outlook.com"
+    },
+
+    "record": {
+        "CNAME": "example.com",
+        "MX": ["mx1.improvmx.com.", "mx2.improvmx.com"], // This is for ImprovMX(if you want custom email like hi@noah.is-cool.dev)
+        "TXT": ["v=spf1 include:spf.improvmx.com ~all"], // Also for ImprovMX
+      "TXT": ["dh=6055d05e33f19d267361f8699ba9c95777d8d62a"], // Discord Connection
+    "proxied": true
```

Comment:
Invalid. One, CNAME is an example record. 2, comments are not supported in JSON. 3. CNAME records must not be used in conjunction with other records and vice versa.